src/vcf/odb_report.cpp

```diff
@@ -73,6 +73,11 @@ namespace ebi
         if (transaction.has_current()) {
             transaction.commit();
         }
+
+        {
+            odb::core::connection_ptr c{db->connection()};
```

Comment:
Creating database connections tends to be an expensive operations. Could you please check if initializing this `connection_ptr` in the constructor can still make any difference in terms of execution time/memory usage?